### PR TITLE
#1: Add specification and design

### LINKED ISSUE #1: Add specification and design
Tasks

- [ ] Add system specification
- [ ] Add design

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #1.

### CODE REVIEW

File: config.ld [hunk 1]
```diff
@@ -0,0 +1,17 @@
+file = "src/main/lua"
```
Comment (morazow) [inline config.ld:1]: This looks like nice way of extending LDoc? :+1:

File: doc/changes/changes_0.1.1.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Exasol Row Level Security (Lua) 0.1.1, released 2022-??-??
+
+Code name:
+
+## Summary
+
+## Features
+
+* ISSUE_NUMBER: description
```
Comment (morazow) [inline doc/changes/changes_0.1.1.md:9]: Please do not forget to update it

File: src/test/java/com/exasol/MetadataReadingIT.java [hunk 3]
```diff
@@ -131,6 +134,43 @@ void testTableFilter() throws SQLException {
...
+            changeVirtualSchemaSource(virtualSchema, sourceSchemaB);
+
+            statement.setString(1, virtualSchema.getName());
+            final ResultSet resultSetB = statement.executeQuery();
+            assertThat(resultSetB, table().row("TB").matches());
+        }
+    }
+
+    void changeVirtualSchemaSource(final VirtualSchema virtualSchema, final Schema sourceSchema)
+    {
```
Comment (morazow) [inline src/test/java/com/exasol/MetadataReadingIT.java:164]:
```suggestion
    void changeVirtualSchemaSource(final VirtualSchema virtualSchema, final Schema sourceSchema) {
```

**Review by morazow — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.1.md (hunk 2 above)
Comment (redcatbear) [reply doc/changes/changes_0.1.1.md:9]: Added.

**Review by morazow — APPROVED** [review]

**Pull request merged**